java-util/src/main/java/io/druid/java/util/common/parsers/ParserUtils.java

```diff
@@ -95,4 +99,15 @@ public static String stripQuotes(String input)
     }
     return input;
   }
+
+  /**
+   * Return a function to generate default column names.
+   * Note that the postfix for default column names starts from 1.
+   *
+   * @return column name generating function
+   */
+  public static IntFunction<String> getDefaultColumnNameGenerator()
```

Comment:
Changed. BTW, the name means "defaultColumnName" generator rather than default "columnNameGenerator".